pyensign/connection.py

```diff
@@ -68,9 +68,9 @@ def publish(self, events):
             yield rep
 
     # TODO: Subscribe should not block
-    def subscribe(self, topics, consumer_id="", consumer_group=None):
+    def subscribe(self, topic_ids, consumer_id="", consumer_group=None):
         open_stream = ensign_pb2.OpenStream(
-            topics=topics, consumer_id=consumer_id, group=consumer_group
+            topics=topic_ids, consumer_id=consumer_id, group=consumer_group
```

Comment:
We are hoping that in the future the user can specify either topic name or topic ID and the server will work both ways; it's just not working that way quite yet.